demo/client/dune

```diff
@@ -2,9 +2,11 @@
  (alias client)
  (target app)
  (enabled_if
-  (= %{profile} "dev"))
+  (and
+   (= %{profile} "dev")
+   (= %{context_name} "melange")))
```

Comment:
not a big deal but I think we can use `(= %{context_name} melange)` and becomes even cleaner :chefs_kiss:

Reply:
nice. updated in https://github.com/ml-in-barcelona/server-reason-react/pull/124/commits/9095b40a920be0d513207a88018e54d786233b05.